GitCommands/Git/GitModule.cs

```diff
@@ -595,6 +596,30 @@ public void SaveBlobAs(string saveAs, string blob)
             stream.Write(blobData, 0, blobData.Length);
         }
 
+        public void SaveTreeAs(string saveAs, string hash)
```

Comment:
This functionality doesn't belong to `GitModule`, it is specific `RevisionFileTreeControl`.

Reply:
Have moved to RevisionFileTreeControl